www/app/app.ts

```diff
@@ -82,6 +82,7 @@ export class App {
     this.rootEl.addEventListener(
         'ShowServerRename', this.rootEl.showServerRename.bind(this.rootEl));
     this.feedbackViewEl.$.submitButton.addEventListener('tap', this.submitFeedback.bind(this));
+    this.rootEl.addEventListener('PrivacyAcked', this.privacyAcked.bind(this));
```

Comment:
We could use a more explicit name like `PrivacyTermsAccepted` for readability.

Reply:
Done  in a2ecc48.